src/store/modules/registry.ts

```diff
@@ -209,8 +209,6 @@ const actions = {
       if (meta) injected.push({ ...meta, injected: true });
     });
     commit('setInjected', injected);
-    await dispatch('account/getBalances', null, { root: true });
-    await dispatch('account/getAllowances', { tokens }, { root: true });
```

Comment:
If we're still using this function, will we not still need to fetch balances/allowances when new tokens are added? Or is that happening elsewhere?

Reply:
that is happening elsewhere and is reactive

Reply:
as even if we fetch balances and allowances via the account store, the values aren't actually used from the store